static/js/components/forms/MenuItemForm.tsx

```diff
@@ -21,28 +21,18 @@ interface Props {
 
 export type MenuItemFormValues = {
   menuItemTitle: string
-  menuItemType: LinkType.Internal | LinkType.External
-  externalLink: string
+  menuItemType: LinkType.Internal
   internalLink: string
 }
 
 const schema = yup.object().shape({
   menuItemTitle: yup.string().required().label("Title"),
-  menuItemType: yup.mixed().oneOf([LinkType.Internal, LinkType.External]),
-  externalLink: yup.string().label("External link").url().when("menuItemType", {
-    is: LinkType.External,
-    then: yup.string().required(),
-  }),
-  internalLink: yup.string().label("Internal link").when("menuItemType", {
-    is: LinkType.Internal,
-    then: yup.string().required(),
-  }),
+  internalLink: yup.string().required().label("Internal link"),
```

Comment:
I think the label should be just `Link` now
<img width="513" alt="image" src="https://github.com/mitodl/ocw-studio/assets/109785089/cee23cdc-ee42-4d28-8aba-9029f3b012fa">

Also, do you think perhaps we should use something else instead of `internalLink`?
Initially the variable name set out distinction for internal links, and external links, but now it is kind of ambiguous.

Even though the "Content" is internal, but content will also contain external resources.
Maybe it could be something like `link` or `contentLink`, or anything else ...